librz/core/ctypes.c

```diff
@@ -305,20 +305,23 @@ static void core_types_struct_print(RzCore *core, RzBaseType *btype, RzOutputMod
 	switch (mode) {
 	case RZ_OUTPUT_MODE_JSON: {
 		rz_return_if_fail(pj);
+		if (!btype) {
+			break;
+		}
```

Comment:
```suggestion
```
This is not needed because few lines above there is `rz_return_if_fail(core && vbtype)` so btype is guaranteed to be non-null at this point.